tests/Unit/ControlSystem/Test_Averager.cpp

```diff
@@ -248,6 +251,28 @@ SPECTRE_TEST_CASE("Unit.ControlSystem.Averager.BadCallToAverageTime",
   averager.average_time(0.0);
 }
 
+SPECTRE_TEST_CASE("Unit.ControlSystem.Averager.EqualityAndSerialization",
+                  "[ControlSystem][Unit]") {
+  Averager<2> averager1(1.0, true);
+  Averager<2> averager2(1.0, true);
+  Averager<2> averager3(0.5, false);
+
+  CHECK(averager1 == averager2);
+  CHECK(averager1 != averager3);
+
+  CHECK(serialize_and_deserialize(averager1) == averager1);
```

Comment:
You should also check that two averagers are (not)equal after an update. Here you're just checking that the default values of most members are the same.

Maybe also add a few `CHECK_FALSE`s as well.